alfalfa_worker/Dockerfile

```diff
@@ -150,17 +149,25 @@ RUN git clone --branch $SUNDIALS_TAG --depth 1 https://github.com/LLNL/sundials.
     && make install \
     && cd .. && rm -rf sundials sundials_build
 
-# install and upgrade setuptools because this is needed for symfit (used in E+ Python)
-#    For Assimulo -- numpy, Cython must match the version in pypoetry.toml
-RUN ln -sf /usr/bin/python3 /usr/bin/python \
```

Comment:
pretty big difference here now that we aren't disabling virtualenvs, so the python that is used is now in a Poetry virtualenv.